tests/skylab/optimus/pr/ValidateOptimus.wdl

```diff
@@ -7,7 +7,6 @@ workflow ValidateOptimus {
 
      input {
          # Optimus output files to be checked
-         File bam
```

Comment:
Why is the bam checking removed from validation??

Reply:
It was meaningless. The count matrices and the metrics are created from the bam and we are checking the differences. Bam files often have timestamps and we usually don't get bam checked because of call-caching.